egs/fisher_swbd/s5/local/chain/run_blstm_6h.sh

```diff
@@ -37,8 +37,9 @@ EOF
 fi
 
 dir=$dir${affix:+_$affix}
+build_tree_train_set=train_nodup
 train_set=train_nodup_sp
-ali_dir=exp/tri5a_ali_nodup_sp
+ali_dir=exp/tri5a_ali
```

Comment:
and also-- should have said this before-- please rename ali_dir to build_tree_ali_dir